namespaces/live.cloud-platform.service.justice.gov.uk/offender-management-test/00-namespace.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: offender-management-test
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "true"
```

Comment:
could this please be changed to 
`cloud-platform.justice.gov.uk/is-production: "false"`

Reply:
addressed